packages/dart/sshnp_flutter/lib/src/presentation/widgets/navigation/app_navigation_rail.dart

```diff
@@ -44,6 +44,7 @@ class AppNavigationRail extends ConsumerWidget {
             child: IntrinsicHeight(
               child: NavigationRail(
                 groupAlignment: -1,
+                minWidth: 97,
```

Comment:
Why 97px specifically?

Reply:
That's the width used in the figma design.

Reply:
So ... OK? I guess. 🤷‍♂️